esp/esp/program/modules/handlers/adminclass.py

```diff
@@ -396,6 +399,19 @@ def coteachers(self, request, tl, one, two, module, extra, prog):
             if cls.conflicts(teacher):
                 conflictinguser = teacher
             else:
+                lastProf = RegistrationProfile.getLastForProgram(teacher, prog)
```

Comment:
If a teacher has a profile for a previous program, but not this program, this gets that old profile and spoofs it as being for this program, but then it doesn't save it (if the old profile is more than 5 days old for some reason). Instead of messing with what is presumably a really old decision, you could just make sure to always save this `lastProf` here by bumping the `lastProf.save()` below out one tab.

Reply:
Actually, it might be better to fix the problem at the source, since it does seem to be a little backwards (it probably makes more sense to not save if the other profile is less than five days old, not more). But just to be sure we can still do my suggestion above as well.

Reply:
@willgearty I did the easy version, bumping the lastProf.save() over, but maybe we want to "fix the problem at the source" and fix the backwards logic in a different PR

Reply:
Sounds good. I'll test this as soon as I get a chance. You should feel free to make that other PR whenever you get a chance. I think the only change that is needed is to change the `<` to a `>` (or vice versa).